fs/parsetime.go

```diff
@@ -0,0 +1,106 @@
+package fs
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Time is a time.Time with an offset, with some more parsing options
+type Time time.Time
+
+// For overriding in unittests.
+var (
+	timeNowFunc = func() time.Time { return time.Now() }
```

Comment:
I had to add this mock so the `TestParseTimeUnmarshalJSON` & `TestTimeScan` could test duration offsets as well

Reply:
This should just be

`timeNowFunc = time.Now`